demo/index.html

```diff
@@ -15,5 +15,12 @@
         </style>
     </head>
     <body>
-    </body>
+
+<div class="wsu-sesquicentennial__background">
+    <div class="wsu-sesquicentennial">
+        <a class="wsu-sesquicentennial__link" href="//150.wayne.edu">
```

Comment:
This can go directly to `https://` since the 150 site forces SSL and some of the sites this is on may not.

Reply:
updated